cves/kernel/CVE-2013-3223.yml

```diff
@@ -84,14 +103,10 @@ fixes_instructions: |
 
   Place any notes you would like to make in the notes field.
 fixes:
-- commit:
-  note:
-- commit:
-  note:
 - commit: ef3313e84acbf349caecae942ab3ab731471f1a1
   note: |
-    Taken from NVD references list with Git commit. If you are
-    curating, please fact-check that this commit fixes the vulnerability and replace this comment with 'Manually confirmed'
+    Taken from NVD references list with Git commit, manually confirmed.
```

Comment:
Just from reviewing other curations I would suggest to make this section pop out more is to add information stating what this commit added and deleted, but in terms of the requirements this is not a need.

Reply:
Will do that